src/bodywork/cli/cli.py

```diff
@@ -92,7 +93,7 @@ def cli() -> None:
     deployment_cmd_parser.add_argument(
         "command",
         type=str,
-        choices=["create", "display", "logs"],
+        choices=["create", "display", "logs", "delete"],
```

Comment:
I think this needs to be `delete-job` as well.

Reply:
Ay, that it does!